core/src/com/unciv/models/ruleset/Difficulty.kt

```diff
@@ -47,7 +47,7 @@ class Difficulty: INamed, ICivilopediaText {
     override fun getCivilopediaTextLines(ruleset: Ruleset): List<FormattedLine> {
         val lines = ArrayList<FormattedLine>()
         lines += FormattedLine("Player settings", header = 3)
-        lines += FormattedLine("{Base Happiness}: $baseHappiness ${Fonts.happiness}", indent = 1)
+        lines += FormattedLine("{Base happiness}: $baseHappiness ${Fonts.happiness}", indent = 1)
```

Comment:
Wasn't the plan that after #5963 and 8311e65120c4dec86ecd51fda1e9dc6a67f0c6e2 we were going to capitalize all these game terms?

Reply:
I didn't care much whether 'Base happiness' or 'Base Happiness' stays in, but cared enough to not leave _both_ in the project (one was untranslatable which is why I found it). So I threw some dice skewing for what I saw elsewhere around the project.

> modify this accordingly

Kudos - making your own decisions is always worthy some respect :grin: